aiida_quantumespresso/workflows/protocols/pw/base.yaml

```diff
@@ -0,0 +1,54 @@
+base:
+    clean_workdir: True
+    kpoints_distance: 0.15
+    kpoints_force_parity: False
+    meta_parameters:
+        conv_thr_per_atom: 0.2e-9
+        etot_conv_thr_per_atom: 1.e-5
+        starting_magnetization: 0.1
```

Comment:
I just checked and I don't this it is being used. This makes me think whether it should be though. I used this in combination with the `force_magnetziation` flag in the `PwBaseWorkChain` protocol to force a non-negative starting magnetization even if all elements should be non-magnetic. If we want to reproduce the reconnaissance SCF input generation than this should definitely be added.

Reply:
If I understand correctly, the current implementation will set the `starting_magnetization` to `0.1` for every element that has `magmom == 0` in case the spin type is `COLLINEAR` and no `initial_magnetization` is provided by the user. So basically `force_magnetization` is always `True`. I'm not sure if this should be the default behavior (I think it's ok), but in any case it should be reported that the magnetization is set automatically.

We should really still understand where these default magmom values come from, though. 🤔  They are all either 5 or 7, and it seems that it's just a matter of looking at whether you have partially filled d (magmom = 5) or f (magmom = 7) orbitals. I suppose there is some logic there, just make it as high as it could possibly be. After all, _"magnetic moments relax down"_. 😉 

This obviously won't always give you the right magnetic moment (e.g. high and low spin depending on the separation of the d-levels of an element in an octahedral environment such as Co in LiCoO2; this approach will only find the high spin state), but I suppose it's fine for a first guess.

Reply:
I remember now, the `starting_magnetization` from the yaml is _not_ being used, but that value has essentially been moved to the `magnetization.yaml` file, where there is a `default_magnetization` key, which is also `0.1`. As you said, this means `force_magnetization=True` by default which I think is fine. If a user needs the opposite at some point we can implement it.

Reply:
yes, will do